app/Models/User.php

```diff
@@ -157,6 +157,10 @@ public function icsTokens(): HasMany {
         return $this->hasMany(IcsToken::class, 'user_id', 'id');
     }
 
+    public function webhooks(): HasMany {
+        return $this->hasMany(Webhook::class);
+    }
+
```

Comment:
Is this surfaced in the API when `/users/{id}` is called? 
Anyone, please resolve if this is the case.

Reply:
Webhooks are not in the response for that endpoint